### PR TITLE
Add calculator layout

### PR DESCRIPTION
**Pull request by piero-vic**
# Add calculator layout
- Delete boilerplate.
- Create `Calculator` component.
- Style `Calculator`.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <button type="button">1</button>
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button className="operation-btn" type="button">+</button>
+      <button className="two-col" type="button">0</button>
+      <button type="button">.</button>
+      <button className="operation-btn" type="button">=</button>
+    </div>
+  );
+}
```
Comment (AshakaE) [inline src/components/Calculator.js:30]: The `Calculator.js` component should be class-based

- [ ] Kindly update code

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```
Comment (AshakaE) [inline src/App.js:10]: The `App.js` component should be class-based

- [ ] Kindly update code

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @piero-vic  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (piero-vic) [reply src/components/Calculator.js:30]: Hi. Thanks for the feedback. I get an error when I try to use class-based components.
Untitled
Is it okay to disable the linters just for that line?

**Review by D4nielJ — APPROVED** [review]
## Hello @piero-vic, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**